nodejs.org/locale/uk/index.md

```diff
@@ -1,23 +1,21 @@
 ---
 layout: index.hbs
 labels:
-  current-version: Current Version
-  download: Download
-  download-for: Download for
-  other-downloads: Other Downloads
-  other-lts-downloads: Other LTS Downloads
-  other-current-downloads: Other Current Downloads
-  current: Current
+  current-version: Поточна версія
+  download: Завантажити
+  download-for: Завантажити для
+  other-downloads: Інші завантаження
+  other-lts-downloads: Інші LTS завантаження
+  other-current-downloads: Інші поточні завантаження
+  current: Поточна
   lts: LTS
-  tagline-current: Latest Features
-  tagline-lts: Recommended For Most Users
-  changelog: Changelog
-  api: API Docs
-  version-schedule-prompt: Or have a look at the
-  version-schedule-prompt-link-text: LTS schedule
+  tagline-current: Найновіші можливості
+  tagline-lts: Рекомандовано для більшості
+  changelog: Список змін
+  api: API
```

Comment:
@ghaiklor @Fishrock123 Yep, I've already thought about it, but when we use “Документація про АРІ” it makes it too wide and breaks layout. So I think it will be okay, if we leave it as is. Everyone will understand that this's docs.

![image](https://cloud.githubusercontent.com/assets/3459374/17032160/86f2bdca-4f80-11e6-8d99-85fe427afe4c.png)